outli.el

```diff
@@ -160,6 +167,12 @@ command."
 (defvar-local outli-heading-char nil
   "Character used to indicate heading depth.  Defaults to commment-start.")
 
+(defvar-local outli-heading-levels nil
+  "Number of outline levels to use. Defaults to ")
+
+(defvar-local outli-headings-reverse nil
+  "Whether to reverse the ordering of headings so that longer headings equate to lower-numbered headings.")
+
```

Comment:
Why configure both config-level keywords and local variable for reverse?  Reverse seems to be used exclusively within the face config code.

Reply:
So that Outline mode gets the levels in levels correct. Things like promote/demote header need that information.

Reply:
It's also used to check whether to reverse the levels in the outline alist.

Reply:
Thinking about this when it's not 3am, yes, outli-headings-reverse should just be a variable you can setq if you need it. I didn't have it in the config to start with, which is why it's ended up as both. I'll revert the code to that version and push.

Reply:
I've removed `outli-headings-reverse' from the config.

Reply:
Let's also make this a defcustom.